lib/helper/Puppeteer.js

```diff
@@ -248,9 +248,15 @@ class Puppeteer extends Helper {
       this.debugSection('Session', 'cleaning cookies and localStorage');
       await this.clearCookie();
     }
-    await this.executeScript('localStorage.clear();').catch((err) => {
-      if (!(err.message.indexOf("Storage is disabled inside 'data:' URLs.") > -1)) throw err;
-    });
+    const currentUrl = await this.grabCurrentUrl();
+
+    if (currentUrl !== 'about:blank') {
```

Comment:
What if this is a page with `chrome:` ? This is also a special page where everything fails :) 
Maybe we should check that protocol is `http(s):` here?